docs/guides/src/docs/asciidoc/_includes/setting-up-the-sample-boot.asc

```diff
@@ -12,30 +12,35 @@ download-url: the URL to download the Spring Security distribution
 
 == Setting up the sample
 
-This section outlines how to setup a workspace within https://spring.io/tools/sts[Spring Tool Suite (STS)] so that you can follow along with this guide. The next section outlines generic steps for how to apply Spring Security to your existing application. While you could simply apply the steps to your existing application, we encourage you to follow along with this guide in order to reduce the complexity.
+This section outlines how to setup a workspace within https://spring.io/tools[Eclipse IDE with Spring Tools] so that you can follow along with this guide. The next section outlines generic steps for how to apply Spring Security to your existing application. While you could simply apply the steps to your existing application, we encourage you to follow along with this guide in order to reduce the complexity.
 
 === Obtaining the sample project
 
-Extract the {download-url}[Spring Security Distribution] to a known location and remember it as _SPRING_SECURITY_HOME_.
+Clone the {gh-samples-url}[Spring Security Samples project] to a known location and remember it as _SPRING_SECURITY_HOME_.
+[WARNING]
+====
+These samples are being migrated to a separate project, however, you can still find
+the not migrated samples in the {gh-old-samples-url}[Spring Security repository].
+====
 
 === Import the {starter-appname} sample application
 
 In order to follow along, we encourage you to import the {starter-appname} sample application into your IDE. You may use any IDE you prefer, but the instructions in this guide will assume you are using Spring Tool Suite (STS).
 
-TIP: The completed sample application can be found at _SPRING_SECURITY_HOME_/samples/{completed-config-type}/{completed-appname}
+TIP: The completed sample application can be found at _SPRING_SECURITY_HOME_/{completed-config-type}/{completed-appname}
 
 * If you do not have STS installed, download STS from https://spring.io/tools
 * Start STS and import the sample application into STS using the following steps:
 ** *File->Import*
 ** *Existing Maven Projects*
 ** Click *Next >*
 ** Click *Browse...*
-** Navigate to the samples (i.e. _SPRING_SECURITY_HOME_/samples/{starter-config-type}/{starter-appname}) and click *OK*
+** Navigate to the samples (i.e. _SPRING_SECURITY_HOME_/{starter-config-type}/{starter-appname}) and click *OK*
```

Comment:
I don't think these directions are correct any longer. The naming convention of the projects has changed as has our build tool of choice.

Reply:
Would it be better if we remove the paragraph that says about the STS and in this line just say that the user should import the project in his favorite IDE?

Reply:
In talks with the team, we decided to create a single how-to section on how to import the projects in different IDEs to be referenced by the documentation. This way we are going to have a single file when we need to change it.

Reply:
Yeah, sounds good 👍